src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -42,96 +41,183 @@ public class TurretSubsystem extends SubsystemBase {
       new DebugEntry<Double>(turretVelocity, "Velocity", this);
 
+  private CANSparkMax pitchMotor;
+  private SingleJointedArmSim pitchSim;
+  private Mechanism2d pitchMech;
+  private MechanismRoot2d pitchRoot;
+  private MechanismLigament2d pitchAngleSim;
+  private SimDeviceSim simPitchEncoder;
+  private SimDouble simPitchPos;
+
+  private PIDController pitchPIDController;
+  private CANcoder m_pitchEncoder;
+  private double pitchPosition;
+  private double pitchVelocity;
+  private Consumer<Double> pitchTestPosition;
+  private GenericEntry pitchKP =
+      configTab.add("Pitch KP", Constants.TurretConstants.TURRET_KP).getEntry();
+  private GenericEntry pitchKI =
+      configTab.add("Pitch KI", Constants.TurretConstants.TURRET_KI).getEntry();
+  private GenericEntry pitchKD =
+      configTab.add("Pitch KD", Constants.TurretConstants.TURRET_KD).getEntry();
+  private DebugEntry<Double> pitchPositionEntry =
+      new DebugEntry<Double>(pitchPosition, "Position", this);
+  private DebugEntry<Double> pitchGoalPositionEntry =
+      new DebugEntry<Double>(0.0, "Goal Position", this);
+  private DebugEntry<Double> pitchVelocityEntry =
+      new DebugEntry<Double>(pitchVelocity, "Velocity", this);
+
   private DebugEntry<Double> tagDistanceEntry =
       new DebugEntry<Double>(0.0, "LastMeasuredTagDistance", this);
 
   private final RobotStateManager robotStateManager;
 
   public TurretSubsystem(RobotStateManager robotStateManager) {
-    turretMotor = new CANSparkMax(Constants.TurretConstants.MOTOR_ID, MotorType.kBrushless);
+    turretMotor = new CANSparkMax(Constants.TurretConstants.TURRET_MOTOR_ID, MotorType.kBrushless);
+    pitchMotor = new CANSparkMax(Constants.TurretConstants.PITCH_MOTOR_ID, MotorType.kBrushless);
 
     // Simulation
     if (Robot.isSimulation()) {
+      // Turret
       turretSim =
           new SingleJointedArmSim(
               DCMotor.getNEO(1),
               4,
               3.5 * 0.1016 * 0.1016 / 3,
               0.1016,
-              -Math.toRadians(Constants.TurretConstants.MAX_TURRET_ANGLE_DEGREES),
-              Math.toRadians(Constants.TurretConstants.MAX_TURRET_ANGLE_DEGREES),
+              -Math.toRadians(Constants.TurretConstants.TURRET_MAX_ANGLE_DEGREES),
+              Math.toRadians(Constants.TurretConstants.TURRET_MAX_ANGLE_DEGREES),
               false,
               0);
       turretMech = new Mechanism2d(4, 4);
-      root = turretMech.getRoot("Root", 2, 2);
+      turretRoot = turretMech.getRoot("Root", 2, 2);
       turretAngleSim =
-          root.append(new MechanismLigament2d("Turret", 2, 0, 5, new Color8Bit(Color.kRed)));
+          turretRoot.append(new MechanismLigament2d("Turret", 2, 0, 5, new Color8Bit(Color.kRed)));
       turretTab.add("Turret", turretMech);
+
+      // pitch
+      pitchSim =
+          new SingleJointedArmSim(
+              DCMotor.getNEO(1),
+              4,
+              3.5 * 0.1016 * 0.1016 / 3,
```

Comment:
What are these numbers?  Can we add a comment, or make them into constants, or both?